app/Http/Controllers/BookingController.php

```diff
@@ -70,15 +70,30 @@ public function create(Request $request)
 
     }
 
-    public function all()
+    public function all(Request $request)
     {
-        $hidden = ['guests', 'updated_at'];
+        $search = $request->input('room_id');
+        $hidden = ['guests', 'room_id', 'booking_id', 'guests', 'updated_at'];
         $date = today()->toDateString();
+        $data = Booking::with(['rooms:id,name'])->whereRelation('rooms', 'room_id', '=', "$search")
+            ->whereDate('end', '>=', $date)->orderBy('start', 'asc')->get()->makeHidden($hidden);
+
+        if ($search) {
+
+            if ($data->isEmpty()) {
+                return response()->json($this->responseService->getFormat(
+                    'No Bookings'));
+            }
+
+            return response()->json($this->responseService->getFormat(
+                'Bookings successfully retrieved by room',
```

Comment:
No need for this different message either, as per the docs it should just be 'Bookings successfully retrieved'